### DISCUSSION TITLE
Integration with HM Wired without CCU

### DISCUSSION THREAD

**Discussion by jeriox** [SukramJ/aiohomematic#1589] [Ideas]
Hey there,

I am currently using the old HM integration with a hardware CCU2. In my experience, the device is really annoying and I would be really happy to get rid of it. Is there any chance that a direct communication with an HMW-LGW _without_ a CCU/RaspberryMatic or similar could be part of the new integration? I guess this should be technically viable as Homegear is doing exactly that? (https://doc.homegear.eu/homegear-homematicwired/abstract.html)

**Comment by SukramJ** [comment]
For me that makes no sense! Integrating communication devices is out of scope, and is already done by CCU, RM or Homegear.
Homegear can be used as CCU replacement for HM devices, and can be integrated with this integration too.
There is no chance that I will spend time on this.

**Reply by jeriox** [reply]
I just like the philosophy of having HA as the single software responsible for managing all smart home connectivity. A good example for that would be the ConBee ZigBee sticks: previously, a separate deCONZ instance was required to manage the communication. The integration maintainers now included the communication directly, so that you can just plug in the USB stick and the HA integration takes care of everything, eliminating the need for additional software. I'm really a fan of that approach.

**Reply by SukramJ** [reply]
That sounds good. Then get the technical documentation from eq-3 (not publicly available, nda needed) and work your way through it, and ideally create a corresponding project on GitHub. Of course, I am also interested in the result.

**Reply by jeriox** [reply]
I am a bit surprised by your passive-aggressive tone and you just closing this issue. I open a suggestion in the "Ideas" category with an approach that both HA integrations are following for other technologies and other projects are following with HM Wired. If you are not open to ideas, why do you have this category in the first place? I would have been happy to help out and contribute to the project, but apparently that is not welcome here...

**Reply by SukramJ** [reply]
>   am a bit surprised by your passive-aggressive tone 

Was stimmt mit dir nicht?

**Reply by jeriox** [reply]
Sorry, vielleicht lags an der Formulierung. Aber "das macht keinen Sinn" und "machs doch selbst" finde ich keinen angemessenen Umgang mit konstruktivem Feedback.

**Reply by SukramJ** [reply]
Ich hab dir vor MONATEN geantwortet!!!!  Und jetzt fängst Du an hier rum zu jammern!

Zu deinem Vorschlag: 
Warum macht es Sinn etwas zu bauen was es schon gibt? Hast du selber aufgezählt.
Was hat das ganze mit einer Integration in HA zu tun? Hier werden CCU und Homegear integriert.

> I guess this should be technically viable as Homegear is doing exactly that?

Was hält dich davon ab selber aktiv zu werden, wenn es doch so einfach scheint? Du hast seit Monaten nicht reagiert, oder selbst mal eine Initiative gezeigt.

> Umgang mit konstruktivem Feedback.

Was hat das mit konstruktivem Feedback zu tun? Wo ist da das Feedback?

Um ehrlich zu sein, sind mir nach deinem heutigen Antwortpost deine weiteren Antworten egal.
Ich werde nichts machen.
